source/options.css

```diff
@@ -34,6 +34,29 @@ li[data-validation] {
 	margin-bottom: 0.3em;
 }
 
+details[open] {
+	--border-left: 4px;
+	border-left: solid var(--border-left) #aaa4;
+	padding-bottom: 1px;
+}
+
+summary {
+	--summary-padding: 10px;
+	background: #aaa4;
+	list-style: none;
+	padding: var(--summary-padding);
+	cursor: pointer;
```

Comment:
I'm not a super fan of the style, but I found the default `details` style inadequate for an "app UI"

<img width="555" alt="Screenshot 3" src="https://user-images.githubusercontent.com/1402241/220903085-d16bb5b3-0bd1-41d7-b0c4-7903ebb46203.png">